lib/ecto_ldap/adapter.ex

```diff
@@ -9,6 +10,15 @@ defmodule Ecto.Ldap.Adapter do
     IO.inspect(preprocess)
     IO.inspect(options)
 
+    {:ok, connection} = Exldap.connect
+    {:ok, search_results} = Exldap.search_field(
+        connection,
+        "ou=users,dc=puppetlabs,dc=com",
+        "mail",
+        'jeff.weiss@puppetlabs.com')
```

Comment:
Super hacky for the moment.  I'm not tied to using `exldap`, it simply provided the fastest path to victory at the moment.  I considered migrating over some of the code from `ldap_wrapper` but it would have taken a little longer to accommodate.